darts/timeseries.py

```diff
@@ -1725,6 +1903,55 @@ def to_json(self) -> str:
         """
         return self.pd_dataframe().to_json(orient='split', date_format='iso')
 
+    def to_csv(self,
+               path_or_buf: Optional[Union[str, TextIO]],
+               sep: str = ",",
```

Comment:
Couldn't we just leverage Pandas' `**kwargs` here and not list/maintain them on our side?